lib/utils.pm

```diff
@@ -2118,8 +2118,7 @@ sub assert_secureboot_status {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 360;
-    $timeout = 1200 if is_aarch64;
+    my $timeout = 1200;
```

Comment:
How long need to wait on power? If don't need 1200s, could we set it as $timeout = xxx if is_power?

Reply:
I don't think it need so many condition to set timeout. As if not need 1200s, when it finished, it will go on next code.